js/functions.js

```diff
@@ -194,7 +194,7 @@ MyMlhDash.prototype.getEl = function (user) {
 	var el = '<tr>';
 	if (user.updated_at){
 		var signUpDate = new Date(user.updated_at);
-		var deadline = new Date("2017-02-15T23:59:59.9999Z");
+		var deadline = new Date("2017-04-06T23:59:59.9999Z");
```

Comment:
It would be nice if this was an environment variable like `APP_ID` and it was passed into the  MLHDash object on creation.

Reply:
We pass the [APP_ID](https://github.com/aimeerpierce/saran/blob/bd01ee29ce667268715256b7c663145c25cf7650/views/index.ejs#L222) in here from environment variables on the back end and then set it in the constructor. We can probably do something similar by just adding `deadline` as a property on `this`.

Reply:
We pass those values from the heroku server [here](https://github.com/ghmeier/saran/blob/master/index.js#L27).